odoo/http.py

```diff
@@ -294,6 +295,20 @@ def get_default_session():
 # config parameter `sessions.max_inactivity_seconds` (default: SESSION_LIFETIME)
 SESSION_LIFETIME = 60 * 60 * 24 * 7
 
+# The default duration (8h) before a session is rotated, changing the
+# session id (also on the cookie) but keeping the same content. Can be
+# set via an optional config parameter `sessions.rotation_seconds`.
+SESSION_ROTATION = 60 * 60 * 8
+
+# After a session is rotated, the session should be kept for a couple of
+# seconds to account for network delay between multiple requests which are
+# made at the same time and all use the same old cookie.
+SESSION_DELETION_TIMER = 15
```

Comment:
At first sight I would put an even longer delay:
There is no arm to keep the previous session 1 or 10 minutes longer, while it's not impossible to receive requests using the previous session after more than 15 seconds because of a network delay

Reply:
> Do you see a scenario where this 15 seconds might be considered too short? 

This is a bit my point, you do not know what other requests might takes more than 15 seconds, while increasing this timeout to 5 minutes or even 10 doesn't cause any issues at all.